Formula/pyqt.rb

```diff
@@ -24,19 +24,20 @@ class Pyqt < Formula
 
   def install
     python = Formula["python@3.9"]
+    qt = Formula["qt"]
     args = %W[
       --target-dir #{prefix}
       --no-make
       --confirm-license
     ]
     system "sip-build", *args
     cd "build" do
-      qt_prefix = Formula["qt"].prefix Formula["qt"].version
+      qt_prefix = qt.prefix(qt.version)
       inreplace "inventory.txt", python.opt_prefix, prefix
-      inreplace "inventory.txt", qt_prefix, prefix
+      inreplace "inventory.txt", /#{Regexp.escape(qt_prefix)}(_\d+)?/, prefix
       inreplace "Makefile", /(?<=\$\(INSTALL_ROOT\))#{Regexp.escape(python.opt_prefix)}/, prefix
-      inreplace "designer/Makefile", /(?<=\$\(INSTALL_ROOT\))#{Regexp.escape(qt_prefix)}/, prefix
-      inreplace "qmlscene/Makefile", /(?<=\$\(INSTALL_ROOT\))#{Regexp.escape(qt_prefix)}/, prefix
+      inreplace "designer/Makefile", /(?<=\$\(INSTALL_ROOT\))#{Regexp.escape(qt_prefix)}(_\d+)?/, prefix
+      inreplace "qmlscene/Makefile", /(?<=\$\(INSTALL_ROOT\))#{Regexp.escape(qt_prefix)}(_\d+)?/, prefix
```

Comment:
Do you think you could upstream a patch that will allow proper customisation of the install prefix?

Right now it seems like the build scripts assume you want to install into the same locations as where Python and Qt are installed, but surely not everyone else wants to do that.

Reply:
There will be a `--scripts-dir` option in future `sip` version.

Reply:
Is there a patch to `sip` we can apply that will allow us to use that option here?

Reply:
No 😟, `sip` maintainer just told me this a few hours ago.

Reply:
Did the `sip` maintainer tell you which version it's planned for?